workspaces/src/worker/impls.rs

```diff
@@ -81,8 +81,8 @@ where
                 deposit.unwrap_or(DEFAULT_CALL_DEPOSIT),
             )
             .await
+            .map(CallExecutionDetails::from)
             .map_err(crate::error::Error::from)
-            .and_then(CallExecutionDetails::from_outcome)
     }
 
```

Comment:
Any reason why you decided to switch the calls? The ordering shouldn't matter, right?

Reply:
Yeah the ordering doesn't matter here, this was more to switch over to the new style of error handling where if a user doesn't care about the error or the transaction details itself, they can just use `ok()`. Before, this was being forced handled via `from_outcome`, but that made the transaction details not available. Now, with `ok()`, the details are available and the user is "forced" to handle it